document/src/test/java/com/ritense/document/web/rest/searchfields/SearchFieldResourceIntegrationTest.java

```diff
@@ -179,6 +187,7 @@ void shouldRespondWith400BadRequestOnSearchFieldPathValidationError() throws Exc
     }
 
     @Test
+    @WithMockUser(username = USERNAME, authorities = FULL_ACCESS_ROLE)
```

Comment:
The others use `{FULL_ACCESS_ROLE}`?

Reply:
Varies. Either notation is fine I think.